src/components/NoResults.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div v-if="results.length < 1" class="mb-3 text-center">
+    <Block v-if="block" class="pt-1">
+      Sorry, we can't find any results
```

Comment:
Actually i think this is better
```suggestion
      Oops, we can't find any results
```